.github/workflows/deploy-staging.yml

```diff
@@ -29,13 +29,34 @@ jobs:
     - name: npm install
       run: npm ci
 
-    - name: Build App
+    - name: Build
       run: npm run build -- --configuration staging
 
+    - uses: actions/upload-artifact@v1
+      with:
+        name: staging
+        path: ./dist
+
+  deploy:
+    name: Deploy
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Clone Repository
+      uses: actions/checkout@v2
+
+    - uses: actions/download-artifact@v1
+      with:
+        name: staging
```

Comment:
Do we need to specify the target folder where the artifact has to be downloaded to?

Reply:
Not sure. Added it just to be explicit, which is never bad.

Reply:
ah ok - check